tinyms/losses.py

```diff
@@ -13,8 +13,46 @@
 # limitations under the License.
 # ============================================================================
 
-from mindspore.nn import loss
+from mindspore.nn import loss, SoftmaxCrossEntropyWithLogits
 from mindspore.nn.loss import *
+from mindspore.nn.loss.loss import _Loss
+from tinyms import Tensor
+from tinyms.common import dtype as mstype
```

Comment:
It's not suggested to import mstype here, you can directly call `import tinyms as ts`.